includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -667,7 +667,7 @@ public function submit_handler() {
 								'username' => ( job_manager_generate_username_from_email() || empty( $input_create_account_username ) ) ? '' : $input_create_account_username,
 								'password' => ( wpjm_use_standard_password_setup_email() || empty( $input_create_account_password ) ) ? '' : $input_create_account_password,
 								'email'    => sanitize_text_field( wp_unslash( $input_create_account_email ) ),
-								'role'     => get_option( 'job_manager_registration_role' ),
+								'role'     => apply_filters( 'submit_job_form_create_account_role', get_option( 'job_manager_registration_role' ), $values, $this ),
```

Comment:
Mind adding documentation for this? I've put it on the next milestone.

Reply:
ahh sorry didn't see this, yeah i'll get an update pushed tomorrow